plugin/src/main/java/com/denizenscript/denizen/nms/interfaces/EntityHelper.java

```diff
@@ -466,4 +467,6 @@ public void startUsingItem(LivingEntity entity, EquipmentSlot hand) {
     public void stopUsingItem(LivingEntity entity) {
         throw new UnsupportedOperationException();
     }
+
+    public abstract void openHorseInventory(PlayerTag player, EntityTag horse);

```

Comment:
Generally NMS methods take in the Bukkit types instead of the Denizen types (I.e. `Player` and `AbstractHorse` in this case)